src/build.d

```diff
@@ -147,10 +182,11 @@ alias lexer = memoize!defineLexer;
 auto defineLexer()
 {
     Dependency dependency = {
+        name: "lexer",
         target: env["G"].buildPath("lexer").libName,
         sources: sources.lexer,
-        deps: stringFiles,
-        name: "(DC) D_LEXER_OBJ %-(%s, %)".format(sources.lexer.map!(e => e.baseName).array),
```

Comment:
Changed `name` to mean the name that the user can give to build this dependency.  The `msg` field can be used to set the message to print when executing the dependnecy.